.github/workflows/integration-cloud-token.yml

```diff
@@ -0,0 +1,34 @@
+name: Integration - cloud token
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - glebashnik/integration-cloud-to-github-action
+  # Uncomment it when reviewed
+  #   - master
+  # pull_request:
+  #   branches:
+  #     - master
+  # schedule:
+  #  - cron: '0 11 * * *'
+
+jobs:
+  integration-cloud-token:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          pip install -e .[dev]
+      - name: Run integration tests
+        env:
+          VESPA_TEAM_API_KEY: ${{ secrets.VESPA_TEAM_API_KEY }}
+          CO_API_KEY: ${{ secrets.VESPA_CLOUD_SECRET_TOKEN }}
+          SD_ZIP_ARTIFACTS: true
```

Comment:
`SD_ZIP_ARTIFACTS` is [Screwdriver-specific](https://docs.screwdriver.cd/user-guide/environment-variables), so we probably won't need it.

Reply:
```suggestion
          VESPA_TEAM_API_KEY: ${{ secrets.VESPA_TEAM_API_KEY }}
          VESPA_CLOUD_SECRET_TOKEN: ${{ secrets.VESPA_CLOUD_SECRET_TOKEN }}
          VESPA_CLIENT_TOKEN_ID: ${{ secrets.VESPA_CLIENT_TOKEN_ID}}
```

Reply:
`CO_API_KEY` is the api-key for cohere.com, which is only used in 1 or 2 of the notebooks.

Reply:
Makes sense, thanks for the info.